src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -911,7 +927,7 @@ def set_deployment_user(user_name, password=None):
     Update deployment credentials.(Note, all webapps in your subscription will be impacted)
     '''
     client = web_client_factory()
-    user = User(location='not-really-needed')
+    user = User()
```

Comment:
So I assume you have verified this change works, right?

Reply:
Doesn't the mock test 'test_set_deployment_user_creds' handle this? I simply saw in the model location is no longer defined thus having it was throwing an error.

Reply:
Mock test doesn't verify the service end behavior, but since this field is gone in the model type, then the change should be fine. Honestly, it has not made any sense to have the location argument here.